packages/website/src/components/ASTEsViewer.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from 'react';
+
+import ASTViewer from './ast/ASTViewer';
+import { isRecord } from './ast/utils';
+import type { ASTViewerBaseProps, SelectedRange } from './ast/types';
+import { TSESTree } from '@typescript-eslint/website-eslint';
+
+function isEsNode(
+  value: unknown,
+): value is Record<string, unknown> & TSESTree.BaseNode {
+  return isRecord(value) && 'type' in value && 'loc' in value;
+}
+
+export const propsToFilter = ['parent', 'comments', 'tokens'];
+
+export default function ASTEsViewer(props: ASTViewerBaseProps): JSX.Element {
```

Comment:
nitpick naming - ESTree instead of `Es`.
Also I think that this would be a bit nicer (putting the "type" at the end of the name)

```suggestion
export default function ASTViewerESTree(props: ASTViewerBaseProps): JSX.Element {
```